static/js/EN/EN.js

```diff
@@ -353,7 +353,7 @@ $(document).ready(function() {
     setDefaults();
     update();
     bc.drawChart();
-    var bound = bc.findComfortBoundary(d, 0.5)
+    // var bound = bc.findComfortBoundary(d, 0.5)
```

Comment:
Can this be removed?

Reply:
that was removed since the variable bound is not utilized later in the code. However, just to be on the safe side I have reverted the change.